browser-extension/mv2/src/background/background.js

```diff
@@ -1260,6 +1260,8 @@ BG.Methods.handleSessionRecordingOnClientPageLoad = async (tab) => {
 
     if (sessionRecordingConfig) {
       sessionRecordingData = { config: sessionRecordingConfig, url: tab.url };
+
+      sessionRecordingData.showWidget = sessionRecordingConfig?.autoRecording?.mode === "custom" ?? false;
```

Comment:
```suggestion
      sessionRecordingData.showWidget = sessionRecordingConfig?.autoRecording?.mode === "custom";
```
This should be enough